postfix-3.9.yaml

```diff
@@ -0,0 +1,216 @@
+package:
+  name: postfix-3.9
+  version: 3.9.0
+  epoch: 0
+  description: Secure and fast drop-in replacement for Sendmail (MTA)
+  copyright:
+    - license: IPL-1.0 OR EPL-2.0
+  dependencies:
+    provides:
+      - postfix=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - cyrus-sasl-dev
+      - icu-dev
+      - libnsl-dev
+      - linux-headers
+      - lmdb-dev
+      - m4
+      - mariadb-connector-c-dev
+      - openldap-dev
+      - openssl-dev
+      - pcre2-dev
+      - perl
+      - postgresql-dev
+      - sqlite-dev
+
+data:
+  - name: _subpackages
+    items:
+      ldap:
+      mysql:
+      pcre:
+      pgsql:
+      sqlite:
+
+pipeline:
```

Comment:
Could you add a test too?

Reply:
to be honest, I don't know how to test postfix package, this is where I'm searching help for, but I'll try for sure.

Reply:
thanks to ChatGPT, I wrote some tests o7